### DISCUSSION TITLE
Phising Website

### DISCUSSION THREAD

**Discussion by denisanjayaaa** [vercel/community#3566] [Bug Report]
### Page to Investigate

https://danaidkagett2023-1.vercel.app/

### Steps to Reproduce

https://danafsk-prioritas.vercel.app/

**Comment by amyegan** [comment]
The site is disabled now. Thanks for reporting.

If you believe that a Vercel deployment contains malicious or illegal content, please email [abuse@vercel.com](mailto:abuse@vercel.com) where the request will be dealt with immediately. When doing this, please provide as much relevant information as possible, taking into consideration our [DMCA policy](https://vercel.com/legal/dmca-policy) and [trademark policy](https://vercel.com/legal/trademark-policy).
